ksql-engine/src/main/java/io/confluent/ksql/planner/plan/StructuredDataSourceNode.java

```diff
@@ -98,27 +98,30 @@
   public StructuredDataSourceNode(
       @JsonProperty("id") final PlanNodeId id,
       @JsonProperty("structuredDataSource") final StructuredDataSource structuredDataSource,
-      @JsonProperty("schema") final Schema schema,
-      @JsonProperty("key") final KeyField keyField
+      @JsonProperty("alias") final String alias
   ) {
-    this(id, structuredDataSource, schema, keyField, MaterializedFactory::create);
+    this(id, structuredDataSource, alias, MaterializedFactory::create);
   }
 
   @VisibleForTesting
   StructuredDataSourceNode(
       final PlanNodeId id,
       final StructuredDataSource structuredDataSource,
-      final Schema schema,
-      final KeyField keyField,
+      final String alias,
       final Function<KsqlConfig, MaterializedFactory> materializedFactorySupplier
   ) {
     super(id, structuredDataSource.getDataSourceType());
-    this.schema = requireNonNull(schema, "schema");
     this.structuredDataSource = requireNonNull(structuredDataSource, "structuredDataSource");
+    this.schema = SchemaUtil.buildSchemaWithAlias(structuredDataSource.getSchema(), alias);
```

Comment:
@agavra I believe this line is the functional change of this PR. Previously, schemas in `StructuredDataSourceNode` were not fully qualified. (The newly added unit test in `StructuredDataSourceNodeTest.java` should fail on the current `master`.) There doesn't appear to be any functional change to `JoinNode`, so that might've been the source of your confusion?

Reply:
I guess what I meant is that it seems like there are no backwards compatibility issues. So internally we store the fully qualified schema, but were you able to access it (top level) using the qualified names beforehand?

Reply:
Ah I see. From manual playing around, it looks like a user is able to access fields using qualified names both before and after this change:
```
CREATE STREAM test (id BIGINT, msg VARCHAR) WITH (...);
SELECT test.id, test.msg from test;
```

Reply:
In the metastore the schema for a source just contains the field names. That hasn't changed.

When building a query there was inconsistency in the schemas stored in source nodes for join and non-join queries.  The former just had field names and the latter had fully qualified names.  This PR standardizes on fully qualified.

This PR also looks to ensure there is no compatibility issues as it ensures everything else works as it did.